firebase-perf/firebase-perf.gradle

```diff
@@ -109,6 +109,8 @@ dependencies {
     // 3rd Party Deps
     implementation 'com.squareup.okhttp3:okhttp:3.12.1'
     compileOnly group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.6'
+    implementation 'com.google.dagger:dagger:2.27'
```

Comment:
doesn't this mean anyone depending on this library has to then pull in dagger?

What if they don't use dagger for their DI?

Reply:
Apparently the underlying dependency of Firebase Performance (Google Transport) already uses Dagger. So, this should not come as a surprise to the developers using Firebase Performance. Hopefully the cost is well justified for a better quality predictable SDK.